README.md

```diff
@@ -58,8 +59,10 @@ repositories {
 dependencies {
     // Other dependencies
     compile group: 'com.cloudant', name: 'cloudant-sync-datastore-core', version:'0.12.1'
-    // include this if you're targeting Android
+    // include this if you're targeting Android and do not want datastore encryption
     compile group: 'com.cloudant', name: 'cloudant-sync-datastore-android', version:'0.12.1'
+    // include this if you're targeting Android and want datastore encryption
+    compile group: 'com.cloudant', name: 'cloudant-sync-datastore-android-encryption', version:'0.12.1'
```

Comment:
That text was quite badly worded and definitely gave that incorrect impression. The `-encryption` jar is not a substitute for `cloudant-sync-datastore-android`, but should be used in addition. Improved wording in 1740b57.